AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/InitialSync/InitialSyncOperationTests.swift

```diff
@@ -111,19 +158,31 @@ class InitialSyncOperationTests: XCTestCase {
         storageAdapter.returnOnQueryModelSyncMetadata(nil)
 
         let reconciliationQueue = MockReconciliationQueue()
-        let syncCallbackReceived = expectation(description: "Sync callback received, sync operation is complete")
         let operation = InitialSyncOperation(
             modelType: MockSynced.self,
             api: apiPlugin,
             reconciliationQueue: reconciliationQueue,
             storageAdapter: storageAdapter,
-            dataStoreConfiguration: .default) { _ in
-                syncCallbackReceived.fulfill()
-        }
+            dataStoreConfiguration: .default)
+
+        let syncCompletionReceived = expectation(description: "Sync completion received, sync operation is complete")
+        let finishedReceived = expectation(description: "InitialSyncOperation finished paginating and offering")
+        let sink = operation.publisher.sink(receiveCompletion: { _ in
+           syncCompletionReceived.fulfill()
+        }, receiveValue: { value in
```

Comment:
probably update name of the test to `testInvokesPublisherCompletion` since it's no longer a completion callback

Reply:
updated